content/project/agile.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "AGILE"
+title: "AGILE-IoT"
 date: 2020-03-11T10:45:46-04:00
 logo: "/images/research/agile.png"
```

Comment:
You've deleted this image as part of this pull request. Is there a plan to replace this image? Otherwise there will be a broken image related to this project.

Reply:
Fixed?